q2cli/commands.py

```diff
@@ -23,13 +23,29 @@ class RootCommand(click.MultiCommand):
         ('dev', q2cli.dev.dev)
     ])
 
+    def __init__(self, *args, **kwargs):
+        # 'help' is `None`, drop it to supply our own.
+        if 'help' in kwargs:
+            del kwargs['help']
+        super().__init__(help=ROOT_COMMAND_HELP, *args, **kwargs)
```

Comment:
I'll fix this in a future PR, we wanted to get this in so ppl can start playing around with it